packages/salesforcedx-vscode-core/src/messages/i18n.ja.ts

```diff
@@ -0,0 +1,459 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * Conventions:
+ * _message: is for unformatted text that will be shown as-is to
+ * the user.
+ * _text: is for text that will appear in the UI, possibly with
+ * decorations, e.g., $(x) uses the https://octicons.github.com/ and should not
+ * be localized
+ *
+ * If ommitted, we will assume _message.
+ */
+export const messages = {
+  channel_name: 'Salesforce CLI',
+  channel_starting_message: '開始しています: ',
+  channel_end_with_exit_code: 'が終了コード %s で終了しました。',
+  channel_end_with_sfdx_not_found:
+    'Salesforce CLI がインストールされていません。https://developer.salesforce.com/tools/sfdxcli からインストールしてください。',
+  channel_end_with_error: 'がエラーで終了しました。%s',
+  channel_end: 'が終了しました。',
+
+  progress_notification_text: '%s を実行しています。',
+
+  notification_successful_execution_text: '%s が正常に終了しました。',
+  notification_canceled_execution_text: '%s がキャンセルされました。',
+  notification_unsuccessful_execution_text: '%s が失敗しました。',
+  notification_show_button_text: '表示',
+  notification_show_in_status_bar_button_text: 'ステータスバーだけに表示',
+  notification_make_default_dev: 'DevHub 組織を認証',
+
+  predicates_no_folder_opened_text:
+    'フォルダが開かれていません。VS Code で Salesforce DX 形式のプロジェクトを開いてください。',
+  predicates_no_sfdx_project_found_text:
+    '開かれたプロジェクトのルートディレクトリに sfdx-project.json ファイルが見つかりませんでした。VS Code で Salesforce DX 形式のプロジェクトを開いてください。',
+
+  task_view_running_message: '[実行しています] %s',
+
+  status_bar_text: `$(x) %s`,
+  status_bar_tooltip: 'クリックしてコマンドをキャンセル',
+
+  force_auth_web_login_authorize_dev_hub_text: 'SFDX: DevHub 組織を認証',
+  force_auth_web_login_authorize_org_text: 'SFDX: 組織を認証',
+
+  parameter_directory_strict_not_available:
+    '必要なメタデータフォルダ "%s" がこのワークスペースに存在しません。',
+
+  parameter_gatherer_enter_file_name: 'ファイル名を入力',
+  parameter_gatherer_enter_dir_name:
+    'フォルダを入力 (Enter で確認またはEsc でキャンセル)',
+  parameter_gatherer_enter_username_name: 'ユーザ名を入力',
+  parameter_gatherer_enter_alias_name:
+    '組織のエイリアスを入力またはデフォルトのエイリアスを使用',
+  parameter_gatherer_enter_custom_url:
+    'カスタムのログイン URL を入力するかデフォルトの URL を使用',
+  parameter_gatherer_enter_scratch_org_expiration_days:
+    'スクラッチ組織の有効日数を入力(1から30まで) またはデフォルト値(7) を使用',
+  parameter_gatherer_enter_project_name: 'プロジェクトの名前を入力',
+  parameter_gatherer_paste_forceide_url: '設定から forceide:// のURLをペースト',
+  parameter_gatherer_paste_forceide_url_placeholder:
+    '設定の forceide:// URL',
+  parameter_gatherer_invalid_forceide_url:
+    "forceide:// 形式のURLが無効です。設定のApex デバッガのページに表示される forceide:// 形式のURLをコピー・ペーストしてください。From your subscriber's org, ",
```

Comment:
Is "From your subscriber's org," intentionally left as English?

Reply:
It's not intended. I'll fix it.